config/webpack.prod.js

```diff
@@ -21,7 +21,7 @@ const webpackMerge = require('webpack-merge');
  * Webpack Constants
  */
 const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
-const API_URL = process.env.API_URL || 'http://api.almighty.io/api/';
+const API_URL = process.env.API_URL || 'http://api.prod-preview.openshift.io/api/';
```

Comment:
@sanbornsen 

1) Locally you can use either localhost or a demo server. 
2) This is not a new env var, it's a rename of API_URL
3) We renamed it because this is not the only API in the world, so we wanted to namespace the variable name. FABRIC8_WIT = Fabric8 Work Item Tracker - what we now called almighty-core